styleguide/derek/examples/layouts/announcements.ejs

```diff
@@ -1,24 +1,22 @@
 <!-- Announcements -->
 <div class="container">
-  <h4>Announcements</h4>
   <div class="row row-eq-height">
     <div class="col-sm-12">
       <%- partial('../../_partials/cards/announcements.ejs', {
-        'type': 'newProduct',
-        'desc': 'This is a New Product. Vivamus quis sem sollicitudin, cursus tellus eget, tempor ex. Suspendisse ut magna condimentum, blandit sem vel, auctor urna. Aenean eu auctor erat. '
```

Comment:
It's available options are listed right above it on the usage page, but for available solutions that makes sense as well. I would rather add it as the title that we have in the backlog to add to the pattern though, I'll note to transition that when that work gets done. For now I'll add back in that line.